examples/webgl_loader_gltf_extensions.html

```diff
@@ -211,6 +211,7 @@
 					spot1.position.set( 10, 20, 10 );
 					spot1.angle = 0.25;
 					spot1.penumbra = 0.75;
+					spot1.decay = 0.0001;
```

Comment:
Let me know if there is any other better way to fix.

Reply:
Maybe changing the `distance` or the `intensity` makes more sense?

Reply:
`distance` won't help as the default is infinite, but is there an `intensity`  value that looks good for all models in this example? It would be nice to use that, as it's physically correct, but may take some tuning for each model.

Reply:
Yeah, trying `intensity` but seems we need tuning for each model...

Reply:
Maybe just swap the spotlight for a second directional light. 😇